docs/README.md

```diff
@@ -3,3 +3,20 @@
 ## Browser support
 
 We align our browser support targets with that of Wagtail. Have a look at the [official documentation](http://docs.wagtail.io/en/latest/contributing/developing.html).
+
+## Python/Django/Wagtail support
+
+Supporting Python versions as defined in `setup.py` classifiers.
+
+Supporting Django versions as defined in the Django project [supported versions](https://www.djangoproject.com/download/#supported-versions).
+
+Supporting Wagtail versions from `1.7`.
```

Comment:
There is a new documentation at https://github.com/wagtail/wagtail/blob/a9b7265784761d7bd9428b9721bfea9486370c30/docs/releases/upgrading.rst, which I think will be published on the docs site in the upcoming 1.12 release. It also contains supported Django and Python versions per release.

Based on this, I think we should amend our support target to be the same as Wagtail's. As of _today_, that means:

- 1.8 LTS
- 1.10 latest -1
- 1.11 latest

The 1.12 release is just around the corner (first RC already done), once it's out that would mean:

- 1.8 LTS
- 1.11 latest - 1
- 1.12 latest (LTS)

Practically, we also have great incentives to support versions used on _our_ projects that use WagtailDraftail, so I think we should just drop 1.7 and 1.9 for now.